index.html

```diff
@@ -13,6 +13,73 @@
   </head>
 
   <body>
-   
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name"><a href="#">Kevin Toro</a></p>
+          </div>
+          <div class="fa-bars-icon">
+            <i class="fa-solid fa-bars"></i>
+          </div>
```

Comment:
[OPTIONAL] I would suggest you to wrap this hamburger menu with `nav` tag as it is a project requirement.
- The hamburger button is on the right, and has a `nav` tag.
* The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

Reply:
I didn't get that clearly. Should I wrap the whole fa-bars-icon  `div` with a `nav` tag ?